cluster.tf

```diff
@@ -192,6 +192,7 @@ resource "google_container_cluster" "primary" {
   dynamic "master_authorized_networks_config" {
     for_each = length(var.master_authorized_networks) > 0 ? [true] : []
```

Comment:
Probably want to also create this block if `var.gcp_public_cidrs_access_enabled` is not null?